core/tracepoint/enable_spec.rb

```diff
@@ -55,11 +55,14 @@ def test; end
       end.enable { event_name.should equal(:line) }
     end
 
-    it 'can accept param but they are initialized as nil within a block' do
+    it 'can accept param within a block but its initialized as nil' do
       event_name = nil
       TracePoint.new(:line) do |tp|
         event_name = tp.event
-      end.enable { |f| f.should equal(nil) }
+      end.enable do |*args|
+        event_name.should equal(:line)
+        args.should == [nil]
```

Comment:
Oh wow. Congrats, that's a Ruby bug :bomb:! There's absolutely no reason for `enable` to yield `nil` here. It should not yield anything, so `args.should == []`. @Atul9, do you prefer filing a bug report, or should I?

Reply:
It was actually really easy to fix, so I filed the report and fixed it in trunk: https://bugs.ruby-lang.org/issues/14057. So ideally this test would be changed to reflect the correct behavior, and we use the guard `ruby_bug` (check other specs with this).
I'm pretty sure you didn't expect so much work for these specs 😆 !

Reply:
yeah, I didn't expect so much work for these specs but I got to learn about TracePoint & its fun to discover a bug